db/models/Flashcard.js

```diff
@@ -0,0 +1,17 @@
+import mongoose from "mongoose";
+import "./Collection";
+
+const { Schema } = mongoose;
+
+const flashcardSchema = new Schema({
+  collectionId: { type: Schema.Types.ObjectId, ref: "Collection" },
+  question: { type: String, required: true },
+  answer: { type: String, required: true },
+  level: { type: Number, required: true },
+  isCorrect: { type: Boolean, required: false },
```

Comment:
"required: false" is not necessarily required as it is the default.
```suggestion
  isCorrect: { type: Boolean },
```

Reply:
All right, I have adjusted it accordingly.